scripts/docker/ompl.Dockerfile

```diff
@@ -32,16 +32,19 @@ RUN apt-get update && \
     apt-get install -y libspot-dev && \
     pip3 install pygccxml pyplusplus
 COPY . /ompl
-WORKDIR /build
+WORKDIR /ompl
+RUN echo $PATH
 RUN cmake \
+        -G Ninja \
+        -B build \
         -DPYTHON_EXEC=/usr/bin/python3 \
         -DOMPL_REGISTRATION=OFF \
-        -DCMAKE_INSTALL_PREFIX=/usr \
-        -G Ninja \
-        /ompl && \
-    ninja update_bindings -j `nproc` && \
```

Comment:
Need to make sure the bindings are compiled in the docker container, should be
```bash
cmake --build build -t update_bindings
```
Before the main build step.

Reply:
Done in abb64301b1baeab4a62152258accf6ca57030676